src/shard.rs

```diff
@@ -601,6 +601,7 @@ impl FileCache {
                 &path,
                 Some(3), // retry twice if fail
             ))?;
+            log::info!("Download complete.");
```

Comment:
Still needed?

Reply:
Yeah, it's the only clear signal that the download is finished

Reply:
Can we maybe have file path too?